### PR TITLE
fix: misspelled words fixed

### PR DESCRIPTION
**Pull request by baimamboukar**

### CODE REVIEW

File: _posts/2019-10-18-projects.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ Our Open Life Science program is based on the [Mozilla Open Leader program](http
 
 - [Open Yeast Engineering Project (OYEP)](https://mozilla.github.io/leadership-training/round-5/projects/#open-yeast-engineering-project-oyep) - Timothy Stiles
 
-    Would you drink a GMO beer? What if you brewed it yourself? The open yeast engineering project (OYEP) at BosLab is an open-ended exploration into yeast engineering. We get together, run experiments, do analyses, and engineer yeast strains to produce flavors, fragrances, and small molecules with the hopes of dextinctifying forgotten flavors or using some resultant strains to produce glowing banana flavored chocolate stout and artificial rosé. 
+    Would you drink a GMO beer? What if you brewed it yourself? The open yeast engineering project (OYEP) at BosLab is an open-ended exploration into yeast engineering. We get together, run experiments, do analysis, and engineer yeast strains to produce flavors, fragrances, and small molecules with the hopes of dextinctifying forgotten flavors or using some resultant strains to produce glowing banana flavored chocolate stout and artificial rosé.
```
Comment (NPDebs) [inline _posts/2019-10-18-projects.md:115]: The word "analyses" actually works perfectly in this context...as it is referring to not just one, but multiple examinations.

File: _posts/2019-10-18-projects.md [hunk 2]
```diff
@@ -122,7 +122,7 @@ Our Open Life Science program is based on the [Mozilla Open Leader program](http
 
 - [Brain Networks in Python](https://mozilla.github.io/leadership-training/round-4/projects/#brain-networks-in-python) - Isla Staden
 
-    The Brain Networks in Python project seeks to provide open source, documented, tested and modular code to investigate the brain as a network. These analyses are reproducible, transparent and accessible to new and expert brain imaging researchers and network neuroscientists. Openly developed, this project will welcome users and contributors from all disciplines. 
+    The Brain Networks in Python project seeks to provide open source, documented, tested and modular code to investigate the brain as a network. These analysis are reproducible, transparent and accessible to new and expert brain imaging researchers and network neuroscientists. Openly developed, this project will welcome users and contributors from all disciplines.
```
Comment (NPDebs) [inline _posts/2019-10-18-projects.md:125]: Same here.

This analysis...(singular).
These analyses...(plural).

File: _posts/2019-10-18-projects.md [hunk 3]
```diff
@@ -208,7 +208,7 @@ Our Open Life Science program is based on the [Mozilla Open Leader program](http
 
 - [Reproducible Research in Ocean Biosciences and Open-science Training](https://mozilla.github.io/leadership-training/round-3/projects/#reproducible-research-in-ocean-biosciences-and-open-science-training) - Hao Ye
 
-    This seminar course is aimed at graduate students who have completed introductory classes in statistics and programming and are thinking about data analyses for their thesis projects. The goal is for students to develop technical skills (e.g. literate programming, version control), while also reproducing statistical analyses from the literature to gain first-hand experience with the data analysis workflow for scientific research.
```
Comment (NPDebs) [inline _posts/2019-10-18-projects.md:211]: You get the idea. 😃

**Review by NPDebs — COMMENTED** [review]
@baimamboukar Please revisit your changes. Thank you. 🚀

**Comment by bebatut** [comment]
I close this PR. Thanks

**Pull request closed without merging**